File: src/main/java/net/thedragonteam/thedragonlib/network/internet/GETResponse.java
package net.thedragonteam.thedragonlib.network.internet;

public class GETResponse {

	public int statuscode = 200;
	public String result = new String();
	public String target = new String();
	public String params = new String();
	public String useragent = new String();
	
	public GETResponse(int code){
		
		statuscode = code;
		result = "";
		target = "";
		params = "";
		useragent = "";
		
	}
	
	public GETResponse(String res){
		
		statuscode = 200;
		result = res;
		target = "";
		params = "";
		useragent = "";
		
	}
	
	public GETResponse(int code, String res){
		
		statuscode = code;
		result = res;
		target = "";
		params = "";
		useragent = "";
		
	}
	
	public GETResponse(int code, String res, String tar){
		
		statuscode = code;
		result = res;
		target = tar;
		params = "";
		useragent = "";
		
	}
	
	public GETResponse(int code, String res, String tar, String par){
		
		statuscode = code;
		result = res;
		target = tar;
		params = par;
		useragent = "";
		
	}
	
	public GETResponse(int code, String res, String tar, String par, String agent){
		
		statuscode = code;
		result = res;
		target = tar;
		params = par;
		useragent = agent;
		
	}
	
}

File: src/main/java/net/thedragonteam/thedragonlib/network/internet/GETResponse.kt
package net.thedragonteam.thedragonlib.network.internet

class GETResponse {

    var statuscode = 200
    var result = ""
    var target = ""
    var params = ""
    var useragent = ""

    constructor(code: Int) {

        statuscode = code
        result = ""
        target = ""
        params = ""
        useragent = ""

    }

    constructor(res: String) {

        statuscode = 200
        result = res
        target = ""
        params = ""
        useragent = ""

    }

    constructor(code: Int, res: String) {

        statuscode = code
        result = res
        target = ""
        params = ""
        useragent = ""

    }

    constructor(code: Int, res: String, tar: String) {

        statuscode = code
        result = res
        target = tar
        params = ""
        useragent = ""

    }

    constructor(code: Int, res: String, tar: String, par: String) {

        statuscode = code
        result = res
        target = tar
        params = par
        useragent = ""

    }

    constructor(code: Int, res: String, tar: String, par: String, agent: String) {

        statuscode = code
        result = res
        target = tar
        params = par
        useragent = agent

    }

}
